src/_includes/svg/hewlett-foundation.svg

```diff
@@ -0,0 +1 @@
5.6,5.5,5.5,0,0,1-5.5-5.6Zm33.6,8.3-4.5-6a4.93,4.93,0,0,0,4.2-5.2,5.43,5.43,0,0,0-1.6-4c-1-1-2.4-1.4-4.5-1.4h-5.3V50.7h3.1V44.1l4.7,6.5h3.9Zm-8.6-8.5V37h2c2.3,0,3.1,1.3,3.1,2.6,0,1.8-1.3,2.7-4,2.7h-1.1ZM356.7,46h5.1l1.8,4.7h3.3l-6.2-16.3-.1-.3H358l-6.3,16.6H355Zm2.6-7.3,1.5,4.4h-3ZM134.7,80.5H114.3V62.6h-4.7v40.5h4.7V84.8h20.4v18.3h4.7V62.6h-4.7Zm25.1-8a16.31,16.31,0,0,0-15.9,16.1c0,8.8,7.1,15.3,15.9,15.3,6.8,0,12.8-3.6,15-10l.1-.4h-4.7l-.1.2a11,11,0,0,1-10.5,6.9c-5.7,0-10.9-4.8-11.3-10.3h27.3V90a16.18,16.18,0,0,0-2.3-9.6,15.35,15.35,0,0,0-13.5-7.9ZM148.5,86.2a11.41,11.41,0,0,1,11.3-9.6,11.61,11.61,0,0,1,11.4,9.6Zm58,11.7-6.8-21h-3.8l-6.7,21L180,73.4l-.1-.2h-4.7l11.6,29.7.1.2h4.8l6.1-18.7,6.2,18.7h4.8l11.7-29.9h-4.6Zm17.4,5.2h4.4V62.6h-4.4Zm24.3-30.6a16.31,16.31,0,0,0-15.9,16.1c0,8.8,7.1,15.3,15.9,15.3,6.8,0,12.8-3.6,15-10l.1-.4h-4.7v.2a11,11,0,0,1-10.5,6.9c-5.7,0-10.9-4.8-11.3-10.3h27.4V90a16.18,16.18,0,0,0-2.3-9.6,15.83,15.83,0,0,0-13.7-7.9ZM236.9,86.2a11.41,11.41,0,0,1,11.3-9.6,11.61,11.61,0,0,1,11.4,9.6Zm54.7-23.5h-4.4V73.3h-12V62.7h-4.4V73.3h-6.3v4.1h6.3v25.8h4.4V77.3h12v25.8h4.4V77.3h7.2V73.2h-7.2Zm236.5,40.4h4.4V72.5h-4.4ZM516.5,61.9h-4.4V72.5h-6.3v4.1h6.3v26.6h4.4V76.6h7.2V72.5h-7.2Zm79.6,13.6c-2.6-2.6-5.6-3.7-9.7-3.7a11.18,11.18,0,0,0-9,4.1V72.6H573v30.6h4.4V86.3c-.1-3.4.9-6.3,2.7-8a9.19,9.19,0,0,1,12.6,0c1.8,1.7,2.8,4.6,2.8,8v16.8h4.4V86.3C600,81.7,598.6,77.9,596.1,75.5Zm-68-6.1h4.4V61.9h-4.4Zm-30.4,7.9c-3.6-5-9.2-5.5-11.5-5.5a16.06,16.06,0,1,0-1.13,32.1h1.13a14.4,14.4,0,0,0,11.5-5.5v4.8h4.4V72.6h-4.4ZM486.2,99.8a11.7,11.7,0,0,1-12-11.39V87.9c0-7.9,6-12.1,12-12.1a11.85,11.85,0,0,1,11.7,12h0a11.66,11.66,0,0,1-11.31,12Zm67-28a16,16,0,1,0,16,16,16.06,16.06,0,0,0-16-16Zm.1,28A11.72,11.72,0,0,1,541.7,88a11.85,11.85,0,0,1,11.57-12.1h0a11.71,11.71,0,0,1,11.6,11.8A11.9,11.9,0,0,1,553.3,99.8Zm-201.5-28a16,16,0,1,0,16.05,16A16.06,16.06,0,0,0,351.8,71.8Zm.1,28A11.72,11.72,0,0,1,340.3,88a11.85,11.85,0,0,1,11.57-12.1h0a11.71,11.71,0,0,1,11.6,11.8,11.85,11.85,0,0,1-11.57,12.1ZM312,103.1h4.7V84.5h15.9V80.3H316.7V66.2h16.5V61.9H312.1l-.1,41.2ZM462.1,77.2c-3.9-5-9.8-5.4-11.5-5.4a16.06,16.06,0,1,0-1.13,32.1h1.13a14.68,14.68,0,0,0,11.5-5.5v4.8h4.4V62h-4.4ZM450.7,99.8a11.7,11.7,0,0,1-12-11.39V87.9c0-7.9,6-12.1,12-12.1a11.85,11.85,0,0,1,11.7,12h0A11.73,11.73,0,0,1,451,99.8ZM393.9,88.7c0,5.3-1.4,7.2-3.1,8.8a8.14,8.14,0,0,1-6.1,2.3,9,9,0,0,1-6.8-2.7c-1.4-1.8-2.1-4.3-2.1-7.7V72.6h-4.4V89.5c0,4.3.9,7.7,2.7,9.9,1.4,1.7,4.6,4.6,10.6,4.6,5.6,0,8.3-3,9.4-4.7v4h4.2V72.7h-4.4v16Zm23.3-16.9a11.18,11.18,0,0,0-9,4.1V72.6h-4.4v30.6h4.4V86.3c-.1-3.4.9-6.3,2.7-8a9.19,9.19,0,0,1,12.6,0c1.8,1.7,2.8,4.6,2.8,8v16.8h4.4V86.3c0-4.7-1.3-8.3-3.8-10.8A12.89,12.89,0,0,0,417.2,71.8Z" transform="translate(0 0)" fill="fill-f00"/><path d="M0,71.2H3.9V.1H0ZM53.3,31H47.5V0H43.7V31H13.3V0H9.4V34.8H43.6v5.8H9.6V71.1h3.9V44.4H43.7V61.5h3.9v-17h5.8V61.6h3.9V.1H53.4V31Zm0,9.7H47.5V34.9h5.8Zm199.9.2V36.6h-3v4.3h-4.3V44h4.3v4.3h3.1V44h4.3V40.9Z" transform="translate(0 0)" fill="fill-f00"/></svg>
```

Comment:
When embedding this svg file in the html markup, it's wrapped in `<svg>` tag, so that the `<svg>` tag is not needed in here. Please remove the `<svg>` wrapper and the `<title>` tag from this file.

Reply:
@cindyli removing the svg wrapper causes the logo to become distorted.

Reply:
Don't worry. I've got it working right.

Reply:
awesome I'll push the other changes

Reply:
I see I had to change the `viewBox` dimensions

Reply:
I see, I had to change the `viewBox` dimensions